data/clients.json

```diff
@@ -128,7 +128,7 @@
     },
     {
         "doap": null,
-        "last_renewed": "2020-05-13T19:46:58",
+        "last_renewed": "2021-01-18T17:43:00",
         "name": "ChatSecure",
```

Comment:
Let's not be too enthusiastic, I am following the iOS developments and I know this is not actively being developed anymore (unfortunately). I would expect the developer to update preferably, if he reconsiders to start over.